packages/core/src/Cardano/util/computeImplicitCoin.ts

```diff
@@ -93,6 +93,15 @@ const computeConwayDeposits = (
       case Cardano.CertificateType.Unregistration:
         if (stakeCredentialInRewardAccounts(cert.stakeCredential, rewardAccounts)) reclaimDeposit += cert.deposit;
         break;
+      case Cardano.CertificateType.RegisterDelegateRepresentative:
+        // REVIEW: assuming a single drep scenario. If this should work in a multiple account scenario,
```

Comment:
@AngelCastilloB , @mkazlauskas, is the context of multi-sig, should I include an "ownDrepId" parameter to the function?

Reply:
I think we need to include it even for single wallets. This is now also used for producing transaction summary and reclaiming a deposit is basically spending user's owned funds. Similarly for registering/depositing: it is relevant to the user how much of the funds in the transaction he remains in control of

Reply:
done here [3a4da4f](https://github.com/input-output-hk/cardano-js-sdk/pull/1069/commits/3a4da4fa07dacc071d0f3cd0e567333f491c261a)